.github/workflows/main.yml

```diff
@@ -39,3 +39,9 @@ jobs:
         run: yarn test
         env:
           CI: true
+      - name: Test Install time
+        run: |
+          cd ../ && mkdir test && cd test
+          yarn cache clean
+          yarn add file:../blitz/packages/blitz --non-interactive | tee out.txt
+          cat out.txt | ../blitz/.github/checkInstallTime.js
```

Comment:
Random question I was wondering though why we need the intermediary file here?

Reply:
You mean `out.txt`? I don't pipe the yarn output directly so it's not swallowed from the logs.